src/main/java/org/mifos/connector/ams/interop/AmsCommonService.java

```diff
@@ -26,7 +26,7 @@
 import static org.mifos.connector.ams.zeebe.ZeebeVariables.TENANT_ID;
 
 @Component
-@ConditionalOnExpression("${ams.local.enabled}")
```

Comment:
ams_local_enable = true,  then go to fineract
ams_local_enable = false, then go to mock api